src/node.cc

```diff
@@ -4213,6 +4213,17 @@ inline void PlatformInit() {
     } while (min + 1 < max);
   }
 #endif  // __POSIX__
+#ifdef _WIN32
+  for (int fd = 0; fd <= 2; ++fd) {
+    auto handle = reinterpret_cast<HANDLE>(_get_osfhandle(fd));
+    if (handle == INVALID_HANDLE_VALUE ||
+        GetFileType(handle) == FILE_TYPE_UNKNOWN) {
+      _close(fd);
```

Comment:
It fails on Windows 2008 and 2012 but succeeds on Windows 10. IMHO the cleanest way is to just assert that `_open` returned proper fd.